core/templates/dev/head/filters.js

```diff
@@ -399,7 +399,7 @@ oppia.filter('stripFormatting', [function() {
     // or is the bold or italics tags.
     var tagRegex = new RegExp(
       '(?!<img.*class=".*(' + whitelistedImgClasses.join('|') +
-      ').*".*>)(?!<b>|<\/b>|<i>|<\/i>)<[^>]+>', 'gm');
+      ').*".*>)(?!<b>|<\/b>|<i>|<\/i>|<p>|<\/p>|<br)<[^>]+>', 'gm');
```

Comment:
Should the br have a closing '>'?

Tests?

Reply:
Just checked it. Both are working fine. I think you are right, closing tag will have a better semantic. I'll include it.

Reply:
Do I have to write an e2e test for it?

Reply:
No, just karma. Prefer Karma tests where possible, e2e tests take ages.